packages/website/src/app/docs/edge/guides/versioning/upgrading/general/page.mdx

```diff
@@ -25,8 +25,8 @@ you have reviewed the changes and understand what steps you need to take.
    {
        inputs = {
            pkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
-   -       panfactum.url = "github:NixOS/nixpkgs/old-version";
-   +       panfactum.url = "github:NixOS/nixpkgs/new-version";
+   -       panfactum.url = github:panfactum/stack/edge.24-10-09
+   +       panfactum.url = "github:panfactum/stack/edge.24-10-15";
```

Comment:
I don't think you meant to commit this?

Reply:
I actually did.

1. I believe the example was incorrect.
2. I think showing a real example is good

Reply:
Even if we were to do this, we would use our templating system so that these are automatically updated and not hardcoded. I think this is a fine idea, but we should discuss how to do it correctly in a separate PR.